parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -200,11 +198,9 @@ runcmd:
 - "echo \"Package: docker-engine\nPin: version {{WrapAsVariable "dockerEngineVersion"}}\nPin-Priority: 550\n\" > /etc/apt/preferences.d/docker.pref"
 - apt_get_update
 - retrycmd_if_failure 20 10 120 apt-get install -y ebtables docker-engine
-- touch /opt/azure/containers/dockerinstall.complete
```

Comment:
let's not mark docker install as complete until all config has been applied (see line below)